org.gnome.Rhythmbox

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id": "org.gnome.Rhythmbox",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "master",
+    "sdk": "org.gnome.Sdk",
+    "command": "rhythmbox",
+    "rename-appdata-file": "rhythmbox.appdata.xml",
+    "rename-desktop-file": "rhythmbox.desktop",
+    "rename-icon": "rhythmbox",
+    "tags": ["nightly"],
+    "desktop-file-name-prefix": "(Nightly) ",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* OpenGL access */
+        "--device=dri",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Browse user's Music directory */
+        "--filesystem=xdg-music",
+        /* Able to communicate with Rhythmbox's own D-Bus services */
+        "--own-name=org.gnome.Rhythmbox3",
```

Comment:
Sorry for the messy line. I did not realize that changing the app-id would fix the issue, point taken for the next one. Thanks for merging it